GPS next_WP
GPS cur_pos

float32 next_vel
float32 cur_vel

float32 next_heading
float32 cur_heading

uint64 time
uint64 gps_time

float32 battery_soc

uint16 drone_priority

float32 ETA_next_WP

uint32 drone_id